starwars/src/components/Character.js

```diff
@@ -0,0 +1,20 @@
+import React from 'react'
+
+function Character(props) {
+    // console.log(props.characterData);
+    const keys = Object.keys(props.characterData);
+    // console.log(keys);
+    const values = Object.values(props.characterData);
+    // console.log(values);
+     
+    return (
+        <div className="character">
+           {keys.map(key => {
+              return <div className="datum">{key}: {props.characterData[key]}</div>
```

Comment:
nice job getting this done! This is a common pattern, so make sure you remember it! Also, have a look at this http://2ality.com/2012/09/expressions-vs-statements.html